etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -143,7 +144,8 @@ public Map<HttpEndpoint, Function<DomainRequest, DomainResponse>> domainRegistra
     @Override
     public String openApiSpecification() throws UnableToReadOpenApiSpecificationException {
         String fileName = "openapi_etor.yaml";
-        return OpenApiReaderImplementation.getInstance().openAsString(fileName);
+        return OpenApiReaderImplementation.getInstance()
+                .openAsString(fileName, StandardCharsets.UTF_8);
```

Comment:
I wouldn't pass in `StandardCharset.UTF_8`.  There's no reason for `.openAsString(...)` to support other charsets.  Instead, hardcode `StandardCharsets.UTF_8` in `OpenApiReaderImplementation#openAsString`.

This applies to the other places where we call `OpenApiReaderImplementation.getInstance()
                 .openAsString(...)`.

Reply:
I had made this change for this PR. Reverted the change, the diff should be a bit smaller now too